### DISCUSSION TITLE
Scalable Workflow Orchestration, but How?

### DISCUSSION THREAD

**Discussion by mark91m12** [Netflix/conductor-community#81] [General]
Hello guys, i'm looking for an HA installation of conductor on Kubernetes but i did not found anything about it. What i want to achieve is to replicates conductor server in order to increase performance. How is it possible to do that without using dynomite? Currently i'm using conductor installed on a k8s cluster in AZURE env, it would be very helpfully a guide step by step or a meeting in order to show you our problem. Thanks in advance and i will leave here my current configuration:

persistence: MySQL,
indexing: elasticsearch (external) version 6.8.0,
conductor version: 3.1.0

config.properties: 

```
conductor.grpc-server.enabled=false
conductor.db.type=mysql
spring.datasource.url=jdbc:mysql://@@@@@@@@.mysql.database.azure.com:3306/conductor?serverTimezone=UTC
spring.datasource.username=****
spring.datasource.password=****
spring.datasource.hikari.maximum-pool-size=100
spring.datasource.hikari.minimum-idle=2
conductor.indexing.enabled=true
conductor.elasticsearch.url=http://es:9200
conductor.elasticsearch.indexPrefix=conductor
loadSample=false
```

**Comment by aravindanr** [comment]
Conductor is designed to be horizontally scalable, so more instances can be added as necessary. For HA, in addition to persistence and indexing, we recommend using a lock implementation (Redis or Zookeeper) as well.
